### PR TITLE
chore(deps): update jest

### PR DESCRIPTION
**Pull request by alexkrolick**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @alexkrolick has signed the CLA. Thanks for the pull request!

**Comment by priyajeet** [comment]
yarn lock diff looks more than it should?

File: src/__mocks__/react-intl.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from '../../scripts/jest/mocks/react-intl-mocks';
```
Comment (priyajeet) [inline src/__mocks__/react-intl.js]: exclude mocks folder from babel compile, within package.json scripts, similar to tests

File: src/__mocks__/react-intl.js [hunk 2]
```diff
@@ -31,7 +33,7 @@ type Props = {
     locale: string,
 };
 
-export const injectIntl = Component => {
+export const injectIntl = (Component: React.ComponentType<any>) => {
```
Comment (priyajeet) [inline src/__mocks__/react-intl.js]: Probably needs the import * as React on top

**Pull request merged**